### DISCUSSION TITLE
Add to "Other Calendars" instead of "My Calendars?"

### DISCUSSION THREAD

**Discussion by JPCCUSA** [derekantrican/GAS-ICS-Sync#321] [Q&A] [answered]
I'm glad I found this solution to syncing my coworkers' Outlook calendars to my Google Calendar.

Unfortunately, @derekantrican's code adds their calendars to the "My Calendars" section. Is it possible to have their calendars show under the "Other Calendars" section?

**Comment by jonas0b1011001** [comment, marked as answer]
No, "Other calendars" are read-only.

**Reply by derekantrican** [reply]
_Was just typing this up_ 😄 

There is no way that we can add the calendar as a "subscribed calendar" (the "Other Calendars" section) and still programmatically add/remove/modify events. The only other way you might be able to do this would be to run the script on a separate Google Account and then share it with your main Google Account. (but that's probably a lot of work for such a simple outcome)

**Reply by JPCCUSA** [reply]
Wonderful. Thanks for the quick answer and possible solution.
